### PR TITLE
QTM-516 - Implements GRID on Header

### PR DESCRIPTION
**Pull request by alizeleal**
https://jirasoftware.catho.com.br/browse/QTM-516

Review:

- [ ] Docs
- [ ] Code Review

### CODE REVIEW

File: static/style.css [hunk 1]
```diff
@@ -80,6 +78,8 @@ body{
     color: #BCBF12;
     font-size: 32px;
     font-family: 'Nunito', sans-serif;
+    display: flex;
+    align-items: end;
```
Comment (MarcosViniciusPC) [inline static/style.css]: Apparently these attributes are not required. You can verify?

File: static/style.css [hunk 2]
```diff
@@ -97,20 +97,19 @@ body{
 
 .HiddenMenu{
     font-weight: bold;
-    height: 46px;
-    padding-right: 72px;
     align-items: center;
     gap: 50px;
```
Comment (MarcosViniciusPC) [inline static/style.css]: Inside the div with the HiddenMenu class there is only one div. Do I need a gap and align here?

File: static/style.css [hunk 3]
```diff
@@ -97,20 +97,19 @@ body{
...
     justify-content: flex-end;
 }
 
 .HiddenMenu ul {
     display: flex;
+    justify-content: start;
```
Comment (MarcosViniciusPC) [inline static/style.css]:
```suggestion
```

File: static/style.css [hunk 4]
```diff
@@ -97,20 +97,19 @@ body{
...
+    gap: 50px;
+    padding-right: 72px;
```
Comment (MarcosViniciusPC) [inline static/style.css]: I believe that a padding of 12 px is enough, what do you think?

File: static/style.css [hunk 5]
```diff
@@ -213,48 +212,57 @@ a:hover {
...
     font-weight: bold;
     justify-content: space-between;
     align-items: center;
 }
 
-.HiddenMenuNavigationItem a {
-    color: #BCBF12;
-
+.HiddenMenuNavigationItem a{
+    color: #CCCCCC;
```
Comment (MarcosViniciusPC) [inline static/style.css:223]: Was the color of the items different from the initial figma prototype?

File: static/style.css [hunk 6]
```diff
@@ -213,48 +212,57 @@ a:hover {
...
     .MobileMenu {
         top: 72px;
         width: 100%;
         transition: 0.3s; 
         justify-content: end;
+
+        text-decoration: none;
     }
     .MobileMenu ul {
+        color: #CCCCCC;
```
Comment (MarcosViniciusPC) [inline static/style.css:249]: .HiddenMenuNavigationItem already applies this color, do I really need to apply it again here?

File: static/style.css [hunk 7]
```diff
@@ -58,18 +52,22 @@ body{
...
+    align-items: center;
+
 }
 
 .VisibleMenu {
     color: #BCBF12;
     font-weight: bold;
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-end;
```
Comment (MarcosViniciusPC) [inline static/style.css]: This case is related to the use case of the class commenting above. justify-content: flex-end; it is only necessary when the class is used in nav. When used in Hide, this attribute is unnecessary. This demonstrates inconsistent use of the class for various purposes and can be difficult to maintain.

File: pages/_app.js [hunk 8]
```diff
@@ -45,43 +45,56 @@ export default class HomePage extends App {
...
+                <nav id="visibleDesktopMenu" className="VisibleMenu ">
+                  <ul className="NavList">
+                    <Hide small xsmall className="VisibleMenu">
+                      <li>
+                        <a href="/cathocomponents">Catho Components</a>
+                      </li>
+                      <li>
+                        <a href="/quantum">Quantum</a>
+                      </li>
+                    </Hide>
```
Comment (MarcosViniciusPC) [inline pages/_app.js]: Using Hide here generates a new "div" with "li" inside. Semantically it doesn't make sense to have a list (ul) with a div that contains several "li" inside. Hiding the list items using media query from a class makes more sense than using the Hide component. Another point, the use of the VisibleMenu class is not clear. For example: Without looking at the css, what does the VisibleMenu class do?? Show the menu? Why does the button element not have the class and Hide does? He understands?

If the Hide component is removed, the
VisibleMenu class will be used in only one scenario, that is, its use becomes more predictable, you know?

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[] Code review

[x] Layout

File: static/style.css (hunk 4 above)
Comment (alizeleal) [reply static/style.css]: Checked Figma again and indeed 12px is enough

File: static/style.css (hunk 5 above)
Comment (alizeleal) [reply static/style.css:223]: image
Yes! The other version was using the wrong color

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review
[x] Layout

I suggest removing color from .MobileMenu ul

**Pull request merged**